src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -111,7 +111,7 @@ private enum EXECTYPE
         // cached metadata
         private _SqlMetaDataSet _cachedMetaData;
 
-        private Dictionary<int, SqlTceCipherInfoEntry> keysToBeSentToEnclave = new Dictionary<int, SqlTceCipherInfoEntry>();
+        private Dictionary<int, SqlTceCipherInfoEntry> keysToBeSentToEnclave;
```

Comment:
Can we assign `keysToBeSentToEnclave` to null explicitly here? Similar to `enclavePackage` and `enclaveAttestationParameters` below.

Reply:
We can but it's totally pointless in terms of the IL and asm produced and at best a waste of space in the code and at worst detrimental. It's a reference type so the default value is null and the runtime guarentees that the value will be the default at instantiation. Adding null will either do nothing (for a reference type) or for a value type cause a code to be added to the ctor by the compiler to write 0 over the existing 0.

Reply:
Of course, we can rely on the default value. I see that we do assign the null value to reference type when we define them in other places, especially when we will instantiate them during the run time. It's not a blocker for this PR.

Reply:
In general I go with the minimal style which I've picked up from the https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md as I've been working with it. I know there's a mix of styles in SqlClient because it's got a long legacy all the way back from 1.0 and hasn't been extensively reformatted.